release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -361,6 +361,25 @@ With this release, you can now provision an {product-title} cluster with Tang-en
 
 For more information, see xref:../installing/install_config/installing-customizing.adoc#installation-special-config-encryption-threshold_installing-customizing[Configuring an encryption threshold] and xref:../installing/install_config/installing-customizing.adoc#installation-special-config-storage-procedure_installing-customizing[Configuring disk encryption and mirroring].
 
+[id="ocp-4-14-new-cert-process"]
+==== Certificates are now handled by the Machine Config Daemon
+
+In previous {product-title} versions, the MCO read and handled certificates directly from machine configuration files. This led to rotation issues and created unwanted situations, like when certificates were stuck behind a paused machine config pool.
+
+With this release, certificates are no longer templated from bootstrap into machine configuration files. Instead, they are put directly into the Ignition object, written onto disk using the controller config, and handled by the Machine Config Daemon (MCD) during regular cluster operation. The certs are then visible using the Controller Config.
```

Comment:
```suggestion
With this release, certificates are no longer templated from bootstrap into machine configuration files. Instead, they are put directly into the Ignition object, written onto a disk using the controller config, and handled by the Machine Config Daemon (MCD) during regular cluster operation. The certs are then visible using the Controller Config.
```

Reply:
I think an a/the is needed before "disk". Also, I was wondering about Controller Config needing to be capitalized? In the docs, I see config controller, lowercase. I'm unfamiliar with this, so I just wanted to ask. 

Also is Ignition an API object that needs backticks?

Reply:
Ignition doesn't need backticks, I think you're right that controller config should either be lowercase or it should be in backticks e.g. `ControllerConfig` resource...this is a new area for me as well 😄